### PR TITLE
feat(task_execution): tag can be specified in cli argument.

### PR DESCRIPTION
**Pull request by tartavull**
task_execution takes an optional parameter,
the tasks tag.
This is useful to make a kubernetes deployment
which only executes a certain kind of tasks.

### CODE REVIEW

**Comment by tartavull** [comment]
@nkemnitz Do you mind reviewing this code Nico?

File: python/neuroglancer/pipeline/task_execution.py [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
 from time import sleep
 import traceback
 
 from neuroglancer.pipeline import TaskQueue, EmptyVolumeException
 from neuroglancer.pipeline import logger
 
+# if an argument is supplied to this script
+# uses it as a tag to lease only certain kind of tasks
+tag = None
+if len(sys.argv) == 2:
```
Comment (nkemnitz) [inline python/neuroglancer/pipeline/task_execution.py]: If someone calls the script with more than one argument, should it silently ignore all arguments (current behavior) or throw a warning / an error?

Comment (nkemnitz) [inline python/neuroglancer/pipeline/task_execution.py]: That was not a rhetorical question - I really don't know how nitpicky I should be.

Comment (tartavull) [inline python/neuroglancer/pipeline/task_execution.py]: good catch. Didn't think of that.
Let me improve this :D

**Comment by tartavull** [comment]
```bash
$ python -m neuroglancer.pipeline.task_execution --help
Usage: task_execution.py [OPTIONS]

Options:
  --tag TEXT  kind of task to execute
  --help      Show this message and exit.
```
```bash
python -m neuroglancer.pipeline.task_execution foo bar   
Usage: task_execution.py [OPTIONS]

Error: Got unexpected extra arguments (foo bar)

```
```bash
$ python -m neuroglancer.pipeline.task_execution --tag DownsampleTask
```
```bash
$ python -m neuroglancer.pipeline.task_execution 
```

**Pull request merged**